relay-general/src/pii/attachments.rs

```diff
@@ -613,7 +589,13 @@ mod tests {
     }
 
     fn utf16le(s: &str) -> Vec<u8> {
-        UTF_16LE.encode(s, EncoderTrap::Strict).unwrap()
+        s.encode_utf16()
+            .map(|u| u.to_le_bytes())
```

Comment:
`.flat_map(|u| u.to_le_bytes().into_iter())` or something along that line should avoid the double collect here…

Reply:
I tried this again and I'm not sure it's possible.  Problem is that `u.to_le_bytes()` creates a new array, which does not live anywhere, so you can't just turn it into an iterator.  By doing the `collect::<Vec<_>>()` you give it somewhere it lives long enough for the actual flatten.